### ISSUE TITLE
`search` does not output Microsoft To Do

### ISSUE THREAD

**Issue by huxuan** [mas-cli/mas#260]
## Your Environment

- `mas version`: 1.6.3
- macOS version (`system_profiler SPSoftwareDataType -detailLevel mini`): 10.15.3

### mas Install Method

- [X] `brew install mas` (homebrew-core)
- [] [mas-cli/tap](https://github.com/mas-cli/homebrew-tap)
- [] `.pkg` installer from [releases](https://github.com/mas-cli/mas/releases)
- [] Built from source
   - Fork/branch: ? (e.g. mas-cli/master)
   - Xcode version: 10.?

## Describe the Bug

> A clear and concise description of what the bug is.

I tried `todo`, `"To Do"`, `Microsoft` as keyword, but it is not listed in the search result.

## To Reproduce

1. Run `mas search Microsoft`

## Expected Behavior

> A clear and concise description of what you expected to happen.

`Microsoft To Do` should be listed in the search result.

## Actual Behavior

> A clear and concise description of what actually happened.

`Microsoft To Do` is not listed in the search result.

The actual id is `1274495053`.

## Screenshots, Terminal Output

> If applicable, add screenshots to help explain your problem.

```
$ mas search Microsoft
...
```

## Additional Context

> Add any other context about the problem here.

> Did it work in a previous version?

No, I just knew mas and really appreciate this awesome tool.

**Comment by nyg** [comment]
My guess is that the issue is in the [StoreSearch](https://github.com/mas-cli/mas/blob/d2802fda23ea2fc0f2875f51a20f51731308e151/MasKit/Controllers/StoreSearch.swift#L32) file. If the `attribute=allTrackTerm` is removed from the API URL the result will appear. The problem is that other somewhat irrelevant results may also appear (e.g. with "slack", going from 5 to 36 results).

I haven't been able to find out to which fields this `attribute=allTrackTerm` restricts the search. What is clear is that with it, the field `trackName` is not searched, which seems kind of strange to me. Could this be a bug with the iTunes API?

Many other softwares are impacted by this (MacFamilyTree brought me here). Here are the other related issues which I believe will be fixed when this one is too. #176, #220, #221, #258

**Comment by huxuan** [comment]
OK, got that. Thanks for the response and explanation. Then I will close this issue due to redundancy.

**Comment by phatblat** [comment]
This is fixed now in mas 1.6.4

```
↪ mas search microsoft
   462054704  Microsoft Word                                                                         (16.37)
   784801555  Microsoft OneNote                                                                      (16.37)
   462058435  Microsoft Excel                                                                        (16.37)
   462062816  Microsoft PowerPoint                                                                   (16.37)
   985367838  Microsoft Outlook                                                                      (16.37)
  1274495053  Microsoft To Do                                                                        (2.17)
  1295203466  Microsoft Remote Desktop                                                               (10.3.10)
```

**Comment by huxuan** [comment]
That's great! Thanks! @phatblat
